contrib/completion/zsh/_docker

```diff
@@ -665,6 +665,7 @@ __docker_container_subcommand() {
         "($help)--volume-driver=[Optional volume driver for the container]:volume driver:(local)"
         "($help)*--volumes-from=[Mount volumes from the specified container]:volume: "
         "($help -w --workdir)"{-w=,--workdir=}"[Working directory inside the container]:directory:_directories"
+        "($help)-it[shortcut for --interactive and --tty]"
```

Comment:
I don't think this would be the right thing to do, as any shorthand flag can be combined in any order (e.g. both `-ti` or `-it` can be used).

The completion script already has support for "short option stacking"; https://github.com/docker/cli/blob/master/contrib/completion/zsh/_docker#L42-L49

But it's disabled by default, because enabling this feature caused some regression for other completion; see the discussion on https://github.com/moby/moby/pull/17124#issuecomment-150518700

Also, related; https://github.com/ohmyzsh/ohmyzsh/issues/6710 and https://github.com/docker/cli/issues/993

Reply:
Thanks, I wasn't aware of that.

We can close this PR, in the meantime I would use it locally for my setup. :)